pkg/features/kube_features.go

```diff
@@ -488,7 +500,9 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CPUManager:                          {Default: true, PreRelease: featuregate.Beta},
 	CPUCFSQuotaPeriod:                   {Default: false, PreRelease: featuregate.Alpha},
 	TopologyManager:                     {Default: false, PreRelease: featuregate.Alpha},
+	LegacyNodeRoleBehavior:              {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
move down to the `features that enable backwards compatibility but are scheduled to be removed` section